src/docs/user/references/Notification-service.adoc

```diff
@@ -213,6 +213,16 @@ request will be triggered.
 | ERROR
 | A task with no execution attempts left is faulty, failed or in error.
 | A task has finished with the an error status
+| `TASK_PROGRESS`
+| Task progress updated
+| INFO
+| Task progress updated to x%
```

Comment:
I wouldn't say X%. It's enough to say that its progress has been updated.
(Do you confirm that we have a task progress ? isn't this only for workflows ? how can the scheduler know that the task is at x% ?)

Reply:
Here I specify the message as displayed and it contains the completion percentage of the task.
It is better so that if this event is triggered multiple times for as task we how the completion value.
This information is passed by the scheduler in the `NotificationData<TaskInfo>` object when the event is triggered.

Reply:
Oh, I didn't know we had this. Great !